source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -22,6 +22,13 @@ add it with the :command:`swupd bundle-add` command as follows:
 
 Current mixing workflow
 ***********************
+Mixer by default runs *all* build commands in a container to ensure the
+correct version of the tooling is being used. This also allows custom mixes
+to automatically perform downstream format bumps when upstream releases
+a format bump. You can still run mixer natively by appending --native to
```

Comment:
It looks like elsewhere in this doc the convention is to add `*`'s around command flags to italicize them.